tensornetwork/backends/jax/jax_backend.py

```diff
@@ -55,10 +56,17 @@ def __init__(self, dtype: Optional[np.dtype] = None) -> None:
     jsp = libjax.scipy
     self.name = "jax"
     self._dtype = np.dtype(dtype) if dtype is not None else None
+    self.jax_precision = get_jax_precision(libjax, precision)
+
+  def configure(self, #pylint: disable=arguments-differ
+                dtype: Optional[np.dtype] = None,
+                precision: Optional[Text] = None):
+    self._dtype = np.dtype(dtype) if dtype is not None else None
+    self.jax_precision = get_jax_precision(libjax, precision)
```

Comment:
Do we really want them to be text? Why not just actually pass in the precision object?

Reply:
To avoid importing `jax` at tensornetwork import. The way this is designed now is to hard-code config arguments into `backend_factory.py`, which is imported at `import tensornetwork`.

Reply:
you're right, we can actuallt remove it!